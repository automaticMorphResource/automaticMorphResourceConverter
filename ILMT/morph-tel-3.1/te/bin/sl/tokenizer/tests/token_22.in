కొవాలా, సీల్సు, ఎకిడ్నా, కంగారూ.. తదితతర జంతుజాలం హాయిగా, సురక్షితంగా ఇక్కడ సంచరిస్తూ ఉంటాయి. 
