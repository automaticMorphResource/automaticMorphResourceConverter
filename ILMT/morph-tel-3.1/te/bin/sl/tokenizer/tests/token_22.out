<Sentence id="1">
1	కొవాలా	unk
2	,	unk
3	సీల్సు	unk
4	,	unk
5	ఎకిడ్నా	unk
6	,	unk
7	కంగారూ..	unk
8	తదితతర	unk
9	జంతుజాలం	unk
10	హాయిగా	unk
11	,	unk
12	సురక్షితంగా	unk
13	ఇక్కడ	unk
14	సంచరిస్తూ	unk
15	ఉంటాయి	unk
16	.	unk
</Sentence>

